www/activities/diary/js/diary.js

```diff
@@ -170,6 +170,33 @@ app.Diary = {
     el.innerText = dateString;
   },
 
+  sendStatistics: async function () {
+    if (app.Settings.get("statistics", "send-data") != true) {
+      return;
```

Comment:
Instead of leaving the function when this setting is `!= true`, I think it would be better and cleaner to only go into the function if it is `== true`. But this is a matter of style, of course.

Reply:
jeah. its my style. I still changed it to "== true".